### DISCUSSION TITLE
Moving SentryOkHttpInterceptor to non-android project

### DISCUSSION THREAD

**Discussion by theawesomechin** [getsentry/sentry-java#2819] [Ideas]
Is it possible to refactor the `sentry-android-okhttp` lib's interceptors to  be an non android lib. I currently would like to use this interceptors in my non-android gradle project and it seems the lib's classes can be refactored to be used by both projects as not many android specific functionality is referenced in the implementation.

**Comment by romtsn** [comment]
Yes, we're gonna do this for the next major version of the sdk (7.+). Please follow this issue for updates https://github.com/getsentry/sentry-java/issues/1783
